cd/scripts/release-controller.sh

```diff
@@ -89,12 +89,11 @@ if ! (echo "$VERSION" | grep -Eq "stable$"); then
   BUILD_DATE=$(date +%Y-%m-%dT%H:%M)
   CONTROLLER_IMAGE_DOCKERFILE_PATH=$CD_DIR/controller/Dockerfile
 
-  DEFAULT_DOCKER_REPOSITORY="public.ecr.aws/aws-controllers-k8s/controller"
+  #TODO: vijat@ to replace u2r4f3v7 with aws-controllers-k8s
+  DEFAULT_DOCKER_REPOSITORY="public.ecr.aws/u2r4f3v7/$AWS_SERVICE-controller"
```

Comment:
For every change in this file we'll have to redeploy a new `cd` Prow image. May I suggest that we hardcode the correct public image repository into this variable now - and we can then override more easily by modifying the environment variables in the Prow job?

Reply:
Sounds good.